client/src/components/App.jsx

```diff
@@ -41,28 +36,9 @@ const App = () => {
       })
   }
 
- if (user) {
   return (
     <div>
-      <Flex w='100vw'>
-        <Link to="/">
-          <Heading as='h1' fontSize='6vh' h='11.5vh' color='brand.400' p={8}>bored tour</Heading>
-        </Link>
-        <Spacer />
-        <Box p={4}>
-        <Menu>
-          <MenuButton padding='1vw' fontSize='2.5vh' color='brand.400' as={Button} rightIcon={<ChevronDownIcon />}>
-          Username
-          </MenuButton>
-          <MenuList fontSize='2.5vh' color='brand.400'>
-            <Link to="/events"><MenuItem>My Events</MenuItem></Link>
-            <Link to="/friends"><MenuItem>My Friends</MenuItem></Link>
-            <Link to="/interests"><MenuItem>My Interests</MenuItem></Link>
-            <Button fontSize='2.5vh' color='brand.400' onClick={logout}> Log Out </Button>
-          </MenuList>
-        </Menu>
-        </Box>
-      </Flex>
+      <Header />
```

Comment:
I didn't use the Header component on purpose because our App homepage has the non-fade raccoons as our banner and all other pages have Header with faded raccoons

Reply:
> I didn't use the Header component on purpose because our App homepage has the non-fade raccoons as our banner and all other pages have Header with faded raccoons

Ah sorry. I looked at them side-by-side and couldn't tell the difference. Could we still have one header file but conditionally render the appropriate raccoon image based on which page it's on?

Reply:
Yeah! I added back the appropriate header into App.jsx on PR #74 but you can go ahead and make the additional header file and I can delete the header changes in PR #74 after you're done!

Reply:
Thanks! PR #77